### PR TITLE
make it so picker does not take up full width

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**
The picker should stay the same width regardless of whether the selection causes other date fields to appear.

Addresses #47

### CODE REVIEW

**Comment by Blackbaud-ToddRoberts** [comment]
holding to review with @Blackbaud-AdamFunderburk

**Comment by Blackbaud-AlexKingman** [comment]
Looks like you'll need to delete a couple visual test baselines to get those tests to pass.

File: src/app/public/modules/date-range-picker/date-range-picker.component.scss [hunk 1]
```diff
@@ -8,9 +8,9 @@
 }
 
 .sky-date-range-picker-form-group {
-  flex: 1 1 auto;
   padding-left: $sky-padding-half;
   padding-right: $sky-padding-half;
+  flex-basis: calc(100% / 3);
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/date-range-picker/date-range-picker.component.scss]: Should we have a mobile version of this style in the `@media` block below?
```
.sky-date-range-picker-form-group {
  flex-basis: 100%;
  ...
```

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/date-range-picker/date-range-picker.component.scss]: `flex-basis` *should* change the vertical height when you have `flex-direction` set to `column`. Strange it's not having that effect though... Anyways, it does work and doesn't need to hold this up. I'm ok with what you've got.

Comment (Blackbaud-ToddRoberts) [reply src/app/public/modules/date-range-picker/date-range-picker.component.scss]: if it's safer to add that we can, doing so doesn't seem to break anything

Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/date-range-picker/date-range-picker.component.scss]: Yeah, lets do it. I think it's better to be more deliberate in this case.

**Pull request merged**